.devcontainer/Dockerfile

```diff
@@ -9,15 +9,14 @@ RUN apt-get -y update && \
     apt-get -y install --no-install-recommends mono-complete nuget && \
     apt-get clean && rm -rf /var/lib/apt/lists/
 
-# Install CMake, Ninja, LLVM/Clang tools
-RUN apt-get -y update && \
-    apt-get install -y cmake && \
-    apt-get install -y ninja-build && \
-    apt-get install -y clang-11 && \
-    apt-get install -y clang-tidy-11 && \
-    apt-get clean && rm -rf /var/lib/apt/lists/
+RUN apt update \
```

Comment:
fyi: left a bunch of general items on my review comment for you to consider. thanks!